sdk/attestation/azure-security-attestation/inc/azure/attestation/attestation_client.hpp

```diff
@@ -226,7 +266,7 @@ namespace Azure { namespace Security { namespace Attestation {
     static AttestationClient Create(
         std::string const& endpoint,
         std::shared_ptr<Core::Credentials::TokenCredential const> credential,
-        AttestationClientOptions options = AttestationClientOptions(),
+        AttestationClientOptions options = AttestationClientOptions{},
```

Comment:
Just so I understand such a change, when do we use one vs the other?

Reply:
Oh, I see the previous discussion here:
https://github.com/Azure/azure-sdk-for-cpp/pull/3654#discussion_r877638614

@antkmsft, @CaseyCarter, and @LarryOsterman - for the uninitiated, can we please state in the SDK guidelines exactly when and where to use `()` vs `{}` and what (if any) subtle semantic differences exist between the two.

Reply:
You always use {}. [See ES.23](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Res-list).